### PR TITLE
Hls multiple captions

### PR DESCRIPTION
**Pull request by masaball**

### CODE REVIEW

File: app/views/master_files/hls_manifest.m3u8.erb [hunk 1]
```diff
@@ -15,7 +15,12 @@ Unless required by applicable law or agreed to in writing, software distributed
 %>
 #EXTM3U
 <% if @master_file.has_captions? %>
-#EXT-X-MEDIA:TYPE=SUBTITLES,GROUP-ID="subs",LANGUAGE="<%= Settings.caption_default.language %>",NAME="<%= Settings.caption_default.name %>",AUTOSELECT=YES,URI="<%= caption_manifest_master_file_path %>"
+<% @master_file.supplemental_file_captions.append(@master_file.captions).each_with_index do |caption, index| %>
```
Comment (cjcolvar) [inline app/views/master_files/hls_manifest.m3u8.erb:18]: It seems like order may be important here given the firs is marked as default below.  Does `DEFAULT` signify anything to the player?  If it does then I'd think we'd need to figure out a way for the user to select which captions is default from within the edit item workflow.

File: app/controllers/master_files_controller.rb [hunk 2]
```diff
@@ -282,6 +282,12 @@ def hls_manifest
 
   def caption_manifest
     return head :unauthorized if cannot?(:read, @master_file)
+    caption_id = params[:c_id]
+    @caption_url = if caption_id == 'master_file_caption'
+                     captions_master_file_path
```
Comment (cjcolvar) [inline app/controllers/master_files_controller.rb:287]: I don't see a test for this case (id = "master_file_caption").  Could you please add one?

**Review by cjcolvar — COMMENTED** [review]
Looking good so far!

File: app/views/master_files/hls_manifest.m3u8.erb (hunk 1 above)
Comment (masaball) [reply app/views/master_files/hls_manifest.m3u8.erb:18]: `DEFAULT` is supposed to designate the caption chosen by the player without user input. I think making the change to the edit item workflow would probably be best as a follow-up ticket/PR?

**Review by cjcolvar — APPROVED** [review]
Looks good.  Agreed that a follow up PR could make the default be user-selectable.

**Pull request merged**